stable/rabbitmq/templates/_helpers.tpl

```diff
@@ -85,3 +85,38 @@ Get the erlang secret.
         {{- printf "%s" (include "rabbitmq.fullname" .) -}}
     {{- end -}}
 {{- end -}}
+
+{{/*
+Return the proper Docker Image Registry Secret Names
+*/}}
+{{- define "rabbitmq.imagePullSecrets" -}}
+{{/*
+Helm 2.11 supports the assignment of a value to a variable defined in a different scope,
+but Helm 2.9 and 2.10 does not support it, so we need to implement this if-else logic.
+Also, we can not use a single if because lazy evaluation is not an option
+*/}}
+{{- if .Values.global }}
+{{- if .Values.global.imagePullSecrets }}
+imagePullSecrets:
+{{- range .Values.global.imagePullSecrets }}
+  - name: {{ . }}
+{{- end }}
+{{- else if or .Values.image.pullSecrets .Values.metrics.image.pullSecrets }}
```

Comment:
`.Values.metrics.image` exists but `.Values.metrics.image.pullSecrets` don't. Can you add it to the values.yaml?